int64 size
---
aruco_msgs/Marker[] markers